xarray/tests/test_dataarray.py

```diff
@@ -122,21 +122,12 @@ def test_struct_array_dims(self):
         """
         # GH837, GH861
         # checking array subraction when dims are the same
-        p_data = np.array([('John', 180), ('Stacy', 150), ('Dick', 200)],
+        p_data = np.array([('Abe', 180), ('Stacy', 150), ('Dick', 200)],
```

Comment:
not a big fan of the name John?

Reply:
I should add a note -- it has to be alphabetical to ensure consistent ordering in pandas before and after 0.24